Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -144,17 +144,26 @@ def __init__(self, session, tdef):
         self.askmoreinfo = False
 
         self.correctedinfoname = u""
+        self.checkpoint_disabled = False
 
     def __str__(self):
-        return "LibtorrentDownloadImpl <name: '%s' hops: %d>" % (self.correctedinfoname, self.get_hops())
+        return "LibtorrentDownloadImpl <name: '%s' hops: %d checkpoint_disabled: %d>" % \
+               (self.correctedinfoname, self.get_hops(), self.checkpoint_disabled)
 
     def __repr__(self):
         return self.__str__()
 
     def get_def(self):
         return self.tdef
 
-    def setup(self, dcfg=None, pstate=None, initialdlstatus=None, wrapperDelay=0):
+    def set_checkpoint_disabled(self, val=True):
```

Comment:
I would rename this parameter to make it more clear what is meant (`val` -> `disabled` or something similar)